src/state/actions/ServiceTypeActions.js

```diff
@@ -32,6 +32,7 @@ export const getAllServiceTypesAction = () => dispatch => {
     .get('/api/service_types')
     .then(res => {
       dispatch({ type: GET_ALL_SERVICE_TYPES_SUCCESS, payload: res.data });
+      console.log(res.data);
```

Comment:
Probably useful for us in development to see what we're getting back, but definitely wanna remove it before we're finished and ready for production!